### PR TITLE
Feat : Initialize Ecom in React and added Navbar

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
It includes :
1) Header - Which can be used everywhere in the app . It consist of Brand Logo , An input search item , add to cart button , wishlist button and login button .

### CODE REVIEW

**Comment by SuyashTiwariCoder** [comment]
react1

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <div className="search__container">
+      <input type="text" placeholder="Type Your Fav Brand"/><i className="fas fa-search"></i>
+    </div>
+
+    <p className="login__details">
+      <a href="#"><i className="fas fa-2x fa-shopping-cart"></i></a>
+      <a href="#"><i className="fad fa-2x fa-box-heart"></i></a>
+      <a href="#"
+        ><button className="btn--login">Login</button></a
+      >
```
Comment (Bhakti199) [inline src/components/Navbar/Navbar.jsx:22]: use react-icons

**Review by Bhakti199 — APPROVED** [review]
This code looks fine to me.
Folder structure is maintained properly.
you may use react icons
This PR is good to merge.

**Review by harshitadaswani — APPROVED** [review]
The code structure of the navbar looks good. Also, I suggest you use `react icons` as they have a wider range of icons to use. Also, initialization can be directly merged to dev as they don't need to be getting reviewed.

**Pull request merged**